.ci/targets/windows_build_and_platform_tests.yaml

```diff
@@ -1,8 +1,9 @@
-
 tasks:
   - name: "prepare tool"
     script: .ci/scripts/prepare_tool.sh
-  - name: "build examples"
+  - name: "build examples (Win32)"
```

Comment:
I don't think these target yaml files need strings around the name

Reply:
Removed.